package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.ts",
   "scripts": {
     "start": "nodemon src/server.ts",
-    "test": "npm test",
+    "test": "mocha -r ts-node/register 'test/**/*.ts'",
```

Comment:
suggestion -> add --reporter dot option for faster running and cleaner report in the console